docs/intro_about.rst

```diff
@@ -56,6 +56,72 @@ matrices, rotation matrices, symmetric matrices, or permutation matrices. When :
++--------------------------------------------------------------------------------------------------------------------+------------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| :ref:`Two-sided Orthogonal <orthogonal>` :cite:`two_sided_ortho:1`                                                 | :math:`\mathbf{Q}_1`         | :math:`\mathbf{Q}_2` | :math:`\begin{cases} \mathbf{Q}_1^{-1} = \mathbf{Q}_1^\dagger \\ \mathbf{Q}_2^{-1} = \mathbf{Q}_2^\dagger \\ \end{cases}`                                                                                                                                       |
++--------------------------------------------------------------------------------------------------------------------+------------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| :ref:`Two-sided Orthogonal with One Transformation <orthogonal>` :cite:`two_sided_ortho_1trans:umeyama`            | :math:`\mathbf{Q}^{\dagger}` | :math:`\mathbf{Q}`   | :math:`\mathbf{Q}^{-1} = \mathbf{Q}^\dagger`                                                                                                                                                                                                                    |
++--------------------------------------------------------------------------------------------------------------------+------------------------------+----------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| :ref:`Two-sided Permutation <permutation>` :cite:`two_sided_permutation:1`                                         | :math:`\mathbf{P}_1`         | :math:`\mathbf{P}_2` | :math:`\begin{cases} [\mathbf{P}_1]_{ij} \in \{0, 1\} \\ [\mathbf{P}_2]_{ij} \in \{0, 1\} \\ \sum_{i=1}^n [\mathbf{P}_1]_{ij} = \sum_{j=1}^n [\mathbf{P}_1]_{ij} = 1 \\ \sum_{i=1}^n [\mathbf{P}_2]_{ij} = \sum_{j=1}^n [\mathbf{P}_2]_{ij} = 1 \\ \end{cases}` |
```

Comment:
Change `:math:`\mathbf{P}_1` to `:math:`\mathbf{P}_1^\dagger` (I changed the table in the paper a little while ago, it makes things simpler to keep the dagger for P1)

Reply:
The issue is fixed in 860b02b.